README.md

```diff
@@ -20,39 +20,39 @@ Things you can do using Concerto:
 - Deserialize (and optionally validate) instances from JSON
 - Instances are JS objects so they are easy to pass around your application
 - Introspect the model using a powerful set of APIs
-- Convert the model to other formats: JSON Schema, XML Schema, Java, Go, Typescript, Loopback, PlantUML using [concerto-tools](https://github.com/accordproject/concerto-tools).
+- Convert the model to other formats: JSON Schema, XML Schema, Java, Go, Typescript, Loopback, PlantUML using [concerto-tools](https://github.com/accordproject/concerto/tree/master/packages/concerto-tools).
 - Import models from URLs
 - Publish your reusable models to any website, including the Accord Project Open Source model repository, hosted at: https://models.accordproject.org
 
 ## Documentation
 
 This mono-repo contains the following packages:
 
-* concerto-core : core library for model management/parsing/validation/serialization
-* concerto-tools : model converters and tools for Concerto model files
-* concerto-cli : command-line interface for Concerto
+* [concerto-core](https://github.com/accordproject/concerto/tree/master/packages/concerto-core) : core library for model management/parsing/validation/serialization
+* [concerto-tools](https://github.com/accordproject/concerto/tree/master/packages/concerto-tools) : model converters and tools for Concerto model files
+* [concerto-cli](https://github.com/accordproject/concerto/tree/master/packages/concerto-cli) : command-line interface for Concerto
 
 # Installation
 
 To install the command-line interface:
 
-```
+```js
```

Comment:
Please only use `js` for JavaSCript code.

Reply:
Removed js in other places